app/decorators/case_contact_decorator.rb

```diff
@@ -70,4 +70,8 @@ def medium_type_icon
       object.medium_type
     end
   end
+
+  def notes
+    object.notes ? object.notes[0..100] + "..." : ""
```

Comment:
@khiga8 I like to use the [truncate](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-truncate) method.